spec/support/shared_examples/nested_controller.rb

```diff
@@ -42,7 +42,8 @@
         end.to change(resource.class, :count).by(1)
       end
 
-      it 'redirects to the new resource' do
+      # FIXME:  this shouldn't be true, in the case of REST resources
+      xit 'redirects to the new resource' do
```

Comment:
Instead of changing this spec, I would suggest opting out of the `:create` example for `audio` and `video`.   I tried to make this flexible on purpose so that we could customize as needed - you can specify which controller examples you want to use:  https://github.com/dpla/primary-source-sets/blob/develop/spec/controllers/audios_controller_spec.rb#L16 and https://github.com/dpla/primary-source-sets/blob/develop/spec/controllers/videos_controller_spec.rb#L16.  If you take out the `:create` param on these lines, you can just define whatever tests you want for the `create` action in `audio_controller_spec` and `video_controller_spec`.  Let me know if this makes sense.

I wrote a couple of `shared_examples` where you can opt in or out by controller action.  It was my attempt to streamline the process of writing controller specs by abstracting out common test patterns - it may or may not turn out to be helpful in the long run.